api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -152,17 +159,46 @@ const (
 
 const KubeObjectProtectionCaptureIntervalDefault time.Duration = 5 * time.Minute
 
+type ResourceCaptureInstance struct {
+	Name string `json:"name,omitempty"`
+
+	//+optional
+	IncludeClusterScopedResources bool `json:"includeClusterScopedResources,omitempty"`
+
+	//+optional
+	IncludedResources []string `json:"includedResources,omitempty"`
+
+	//+optional
+	ExcludedResources []string `json:"excludedResources,omitempty"`
+
+	//+optional
+	LabelSelector metav1.LabelSelector `json:"labelSelector,omitempty"`
+}
+
+type ResourceRestoreInstance struct {
```

Comment:
I'm leaning toward `Recovery` instead of `Restore`.  I suggest: `ResourceRecoveryGroupSpec`

Reply:
Looks like `GroupSpec` has been appended, but `Restore` remains.  I prefer `Recovery` to align with disaster recovery.  Thoughts?

Reply:
That's an oversight. I'll correct this.